utils/src/fhir-migration.ts

```diff
@@ -0,0 +1,200 @@
+import * as dotenv from "dotenv";
```

Comment:
I figured this file we would only run once so i copied and pasted logic that we had in the api into here to save time